package.json

```diff
@@ -95,14 +95,20 @@
     "@leafygreen-ui/tooltip": "6.3.1",
     "@leafygreen-ui/typography": "8.1.0",
     "@rjsf/core": "4.1.0",
+    "analytics": "link:./src/analytics",
```

Comment:
I would advocate removing `config/modules.js` since we're using this link format now. The webpack aliases introduced there have the same purpose as these.

Reply:
I removed the baseUrl parameter from our tsconfig which is used in the `config/modules.js` to generate the webpack config since I'll like to avoid making changes to our build system until we redo it.